python/tvm/contrib/hexagon/session.py

```diff
@@ -393,3 +394,37 @@ def _aot_executor_from_factory(
             remote_file_path = self.upload(binary_path, binary_name)
 
         return self.get_aot_executor(remote_file_path)
+
+
+def create_session(
```

Comment:
Currently, all calls to `create_session` pass arguments owned by a `HexagonLauncher` instance.  The `_workspace` and `_rpc_info` objects are marked as `_`, so these would be expected to be internal implementation details, not part of the external API of the `HexagonLauncher`.

Do we expect to call `create_session` based on something in a workflow that doesn't include `HexagonLauncher`?  If yes, then both the free function `hexagon.session.create_session` and the method `HexagonLauncher.create_session` should exist, with `HexagonLauncher.create_session` calling the free function instead of requiring users to access private variables.  If no, then only the `HexagonLauncher` method should exist.

Reply:
hexagon.session only needs a workspace on the remote device and RPC info. Considering that, it's always dependent to HexagonLauncher. It might be better to just move the create_session to HexagonLauncher?

Reply:
I moved create_session to launcher since it is very dependent to launcher attributes